plugins/org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/model/JavaFileAccess.xtend

```diff
@@ -105,19 +103,20 @@ class JavaFileAccess extends TextFileAccess {
 		true
 	}
 	
-	override getContent() {
-		val sortedImports = Lists.newArrayList(imports.values)
-		Collections.sort(sortedImports)
-		return '''
-			�codeConfig.fileHeader�
-			package �javaType.packageName��IF appendSemicolons�;�ENDIF�
-			
-			�FOR importName : sortedImports�
-				import �importName��IF appendSemicolons�;�ENDIF�
-			�ENDFOR�
-			
-			�internalContents�
-		'''
+	override getContent() '''
+		�codeConfig.fileHeader�
+		package �javaType.packageName��IF appendSemicolons�;�ENDIF�
+
+���		remove duplicates in required imports and sort them			
```

Comment:
I think this is pretty clear from the line below. But if you think it is worth pointing out, please extract a method instead of adding a comment.